<launch>
  <arg name="initial_package" default="robosherlock" />
  <param name="initial_package" type="string" value="$(arg initial_package)" />
  <node name="json_prolog" pkg="json_prolog" type="json_prolog_node" cwd="node" output="screen" />
</launch>
